Resources/DoctrineMigrations/pdo_sqlite/Version20200722150146.php

```diff
@@ -0,0 +1,345 @@
+<?php declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200722150146 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'sqlite', 'Migration can only be executed safely on \'sqlite\'.');
+
+        $this->addSql('CREATE TABLE form_verification (id CHAR(36) NOT NULL --(DC2Type:uuid)
+        , value VARCHAR(255) NOT NULL, code VARCHAR(255) NOT NULL, created DATETIME NOT NULL --(DC2Type:datetime_immutable)
+        , expiration_date DATETIME NOT NULL --(DC2Type:datetime_immutable)
+        , PRIMARY KEY(id))');
+        $this->addSql('DROP INDEX tuple_index');
+        $this->addSql('DROP INDEX IDX_6D6315CC1A445520');
+        $this->addSql('CREATE TEMPORARY TABLE __temp__revision AS SELECT id, content_type_id, created, modified, auto_save_at, deleted, version, start_time, end_time, draft, lock_by, auto_save_by, lock_until, labelField, finalized_by, sha1, deleted_by, finalized_date, raw_data, auto_save, circles, ouuid FROM revision');
+        $this->addSql('DROP TABLE revision');
+        $this->addSql('CREATE TABLE revision (id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, content_type_id BIGINT DEFAULT NULL, created DATETIME NOT NULL, modified DATETIME NOT NULL, auto_save_at DATETIME DEFAULT NULL, deleted BOOLEAN NOT NULL, version INTEGER DEFAULT 1 NOT NULL, start_time DATETIME NOT NULL, end_time DATETIME DEFAULT NULL, draft BOOLEAN NOT NULL, lock_by VARCHAR(255) DEFAULT NULL COLLATE BINARY, auto_save_by VARCHAR(255) DEFAULT NULL COLLATE BINARY, lock_until DATETIME DEFAULT NULL, labelField CLOB DEFAULT NULL COLLATE BINARY, finalized_by VARCHAR(255) DEFAULT NULL COLLATE BINARY, sha1 VARCHAR(255) DEFAULT NULL COLLATE BINARY, deleted_by VARCHAR(255) DEFAULT NULL COLLATE BINARY, finalized_date DATETIME DEFAULT NULL, raw_data CLOB DEFAULT NULL COLLATE BINARY --(DC2Type:json_array)
+        , auto_save CLOB DEFAULT NULL COLLATE BINARY --(DC2Type:json_array)
+        , circles CLOB DEFAULT NULL COLLATE BINARY --(DC2Type:simple_array)
+        , ouuid VARCHAR(255) DEFAULT NULL COLLATE utf8_bin, CONSTRAINT FK_6D6315CC1A445520 FOREIGN KEY (content_type_id) REFERENCES content_type (id) NOT DEFERRABLE INITIALLY IMMEDIATE)');
```

Comment:
ouuid VARCHAR(255) DEFAULT NULL COLLATE utf8_bin must be replaced by ouuid VARCHAR(255) DEFAULT NULL COLLATE BINARY

Reply:
fixed b771de0c